src/assets/fontello/LICENSE.txt

```diff
@@ -3,7 +3,7 @@ Font license info
 
```

Comment:
Fontello stuff doesn't really need PRed. I regenerated it to add the calendar icon.